WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1692,7 +1692,7 @@ - (void)uploadMedia:(Media *)media trackingId:(NSString *)mediaUniqueId
             [self.editorView replaceLocalVideoWithID:mediaUniqueId
                                       forRemoteVideo:media.remoteURL
                                         remotePoster:media.thumbnailLocalURL
-                                          videoPress:media.shortcode];
+                                          videoPress:media.videopressGUID];
```

Comment:
This will need a change in the editor pod. Will be done on another PR on the editor.